sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -1156,3 +1156,27 @@ directive:
   transform: >
     $.put.responses["201"].headers["x-ms-content-crc64"]["x-ms-client-name"] = "ContentCrc64";
 ```
+
+### Prepend Blob prefix to service property types
+``` yaml
+directive:
+- from: swagger-document
+  where: $.definitions
+  transform: >
+    $.Logging["x-ms-client-name"] = "BlobAnalyticsLogging";
+    $.Logging.xml = { "name": "Logging"};
```

Comment:
Please remove https://github.com/Azure/azure-sdk-for-net/blob/93d277ac26344ebebc03a0b2ab6ddb0c5735ba69/sdk/storage/Azure.Storage.Blobs/swagger/readme.md#logging since it'll no longer fire.